employees/tests/test_unit_report_model.py

```diff
@@ -255,3 +256,65 @@ def test_that_added_task_activity_must_be_accepted_as_correct_input(self):
         test_activity_type.full_clean()
         test_activity_type.save()
         self.field_should_accept_input("task_activities", TaskActivityType.objects.get(name="test"))
+
+
+class TestGetProjectOrderedByLastReportCreationDate(TestCase):
+    def setUp(self):
```

Comment:
personally I would prefer having projects build with `ProjectFactory`, i.e
```python
project1 = ProjectFactory()
```
and then have reports named like
```python
self.project1_report1 = ReportFactory(author=self.user, project=self.project1)
```
this way the tests would be easier to grasp in my opinion.

Reply:
:+1: done, more code, but as you have said, easier to read :)